.travis.yml

```diff
@@ -0,0 +1,37 @@
+sudo: false
+
+language: cpp
+
+compiler:
+  - gcc
+  - clang
+
+notifications:
+  email: false
```

Comment:
@chschulte Also, if you want to be notified by email upon build failures, this can be done here.

Reply:
Ok, I see.